sentry-opentelemetry/sentry-opentelemetry-bootstrap/src/main/java/io/sentry/opentelemetry/SentryWeakSpanStorage.java

```diff
@@ -0,0 +1,49 @@
+package io.sentry.opentelemetry;
+
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.context.internal.shaded.WeakConcurrentMap;
+import io.sentry.IScopes;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
```

Comment:
We should update the class comment here at some point

Reply:
Yeah